ProcessLib/SmallDeformation/SmallDeformationFEM.h

```diff
@@ -272,6 +296,24 @@ class SmallDeformationLocalAssembler
             auto const& b = _process_data.specific_body_force;
             local_b.noalias() -=
                 (B.transpose() * sigma - N_u_op.transpose() * rho * b) * w;
+            if (_process_data.nonequilibrium_sigma)
+            {
```

Comment:
Possible code duplication?

Reply:
Yes. Removing, and as suggested by @wenqing storing in the ip_data.